content/src/ports/denylist.ts

```diff
@@ -1,50 +1,104 @@
+import { IBaseComponent } from '@well-known-components/interfaces'
 import { resolve } from 'path'
 import { createInterface } from 'readline'
+import { URL } from 'url'
 import { EnvironmentConfig } from '../Environment'
 import { AppComponents } from '../types'
+import { FSComponent } from './fs'
 
-export interface DenylistComponent {
```

Comment:
The convention for ports is having Components defined, we should define them all in a similar way: https://github.com/decentraland/catalyst/blob/main/content/src/ports/bloomFilter.ts#L3

Reply:
Please let me know if there anything in particular to fix in the docs. I'm not sure what convention are you talking about here

Reply:
Nothing wrong with the docs. We're discussing if there's a convention to add the keyword `Component` at the end of every component or not.